lessons/basics/strings.md

```diff
@@ -69,36 +69,36 @@ iex> String.graphemes string
 
 Let's review some of the most important and useful function the String module has for us.
 
-##### `length/1`
+##### Length
 
-Returns the number of Graphemes in the string.
+Returns the number of Graphemes in the string. Function definition is `length/1`.
```

Comment:
That works for me @jeanmatheussouto.  Could you end each of them with a colon?

```
Function definition is `length/1`:
```

Then this is good to merge 👍